examples/dds/fairmq-start-ex-dds.sh.in

```diff
@@ -25,57 +25,56 @@ plugin=${1:-localhost}
 
 exec 5>&1
 output=$(dds-session start | tee >(cat - >&5))
-export DDS_SESSION_ID=$(echo ${output} | grep "DDS session ID: " | cut -d' ' -f4)
-echo "SESSION ID: ${DDS_SESSION_ID}"
+export FAIRMQ_DDS_SESSION_ID=$(echo ${output} | grep "DDS session ID: " | cut -d' ' -f4)
+echo "SESSION ID: ${FAIRMQ_DDS_SESSION_ID}"
```

Comment:
See https://github.com/FairRootGroup/DDS/commit/2814ff33af62db684692542899692720ffdf6a8f.

Reply:
Reverted it back to the old name.